src/Gui/MainWindow.h

```diff
@@ -140,8 +140,6 @@ class GuiExport MainWindow : public QMainWindow
     static MainWindow* getInstance();
     /** Starts the splasher at startup. */
     void startSplasher();
-    /** Stops the splasher after startup. */
-    void stopSplasher();

```

Comment:
You can move the method `void stopSplasher()` back to its original place since you changed the timer to a modern functor-based connection. No need to place it under Q_SLOTS.

Reply:
Ok, done.